documentation/slate/source/includes/realtime.md

```diff
@@ -284,6 +286,55 @@ At the deleted stop area, the departure time of the train with a reduced service
 If "data_freshness" is "base_schedule", then the depature time is displayed.  
 In that case, a link to this disruption can be found in the section "display_informations" for departures, in the "date_times" object itself for stop_schedules.
 
+## <a name="NO_SERVICE"></a>No service
+
+![image](no_service.png)
+
+``` shell
+# Extract of an impacted trip from /disruptions
+{
+    "application_periods": ⊖[
+       ⊖{
+            "begin": "20191210T183100",
+            "end": "20191210T194459"
+        }
+    ],
+    "impacted_objects": ⊖[
+       ⊖{
+            "pt_object": ⊖{
+                "embedded_type": "trip",
+                "id": "OCE:BOA017534OCESNF-20191210",
+                "name": "OCE:BOA017534OCESNF-20191210",
+                "trip": ⊕{2 items}
+            }
+        }
+    ],
+    "severity": ⊖{
+        "effect": "NO_SERVICE"
+        "name": "trip canceled",
+    },
+}
+```
+The effect of the disruption is `NO_SERVICE`. It means that the train won't be circulating at all.
+
+In the disruption, the deleted trip can be found in the list of "impacted_objects" with the
+"application_periods" describing the period(s) of unavailability for the trip.  
+See the [disruption](#disruption) objects section for its full content and description.
+
+<div></div>
+### Journeys
+
+If the trip deleted is used in part by a section of the `base_schedule` journey, Navitia will compute a different
+itinerary to the requested station in `realtime` (without using that trip).  
```

Comment:
I'm not sure this makes sense to me. Do you have an example to illustrate the case ?

Reply:
Ex: You plan a base-schedule journey from La Défense to Chatelet in Paris.
That journey uses a trip (VJ) from line A.
If that very trip is deleted (in realtime), when computing a realtime journey, you will use a different itinerary: either a later trip from line A, or line 1 if it becomes better.

Hope it clarifies the case. If you think of a better way to explain it, glad to add/change it !